main.tf

```diff
@@ -89,6 +94,12 @@ locals {
   public_cidr_blocks = slice(local.subnet_cidr_blocks, 0, local.num_of_azs)
   ## Match the private subnet var to the list of cidr blocks
   private_cidr_blocks = slice(local.subnet_cidr_blocks, local.num_of_azs, length(local.subnet_cidr_blocks))
+  ## Determine cidr blocks for pod network
+  base_pod_cidr_network_bits = tonumber(regex("[^/]*$", var.pod_cidr))
+  pod_cidr_blocks = var.use_pod_cidr == false ? [] : cidrsubnets(
```

Comment:
nit `!var.use_pod_cidr ?`?

Reply:
Your other question made me notice I'm checking for `use_pod_cidr != null` in one part too. Both of these changed.